build/AzurePipelinesTemplates/WindowsAppSDK-BuildFoundation-job.yml

```diff
@@ -19,10 +19,14 @@ parameters:
     type: boolean
     default: true
   - name: "enableLicenseInstall"
+    type: boolean
     default: true
   - name: "isWeeklyRun"
     type: boolean
     default: False
+  - name: "solutionPath"

```

Comment:
Do we need this to be a parameter? Will ever be anything other than WindowsAppRuntime.sln?

Reply:
This is a hold-over from release/1.2-stable/build/ProjectReunion-BuildFoundation.yml and WindowsAppSDK-BuildProject-Steps.yml. As you can see below, it was used for other stuff back then. Hence, I'm deferring the cleanup of this parameter in case we need to lean further back to the old 1.2-stable pattern to unblock PREFast.

  - template: AzurePipelinesTemplates\WindowsAppSDK-BuildProject-Steps.yml
    parameters:
      solutionPath: 'dev\Bootstrap\CS\Microsoft.WindowsAppRuntime.Bootstrap.Net\Microsoft.WindowsAppRuntime.Bootstrap.Net.csproj'